src/test/cypress/cypress/integration/FeedTests.spec.js

```diff
@@ -61,15 +64,16 @@ describe ('FeedScreen tests',function () {
         cy.get('of-light-card').eq(0).click()
             .find('[id^=opfab-feed-light-card]')
             .should('have.class', 'light-card-detail-selected')
-            .should('have.css', 'margin-left','20px')
+            .should('have.css', 'margin-left', '20px')
             .invoke('attr', 'data-urlId')
             .as('firstCardUrlId')
             .then((urlId) => {
-                cy.hash().should('eq', '#/feed/cards/'+urlId);
+                cy.hash().should('eq', '#/feed/cards/' + urlId);
                 cy.get('of-card-details').find('of-detail');
             });
+        cy.get('#opfab-feed-card-not-found').should('not.exist');
 
-
+        tryToLoadNonExistingCard();
```

Comment:
I think it is not necessary to have this test as it's redundant with tests in translation.spec.ts

Reply:
I did 2 separate tests in 2 different places that have different goals : 
* the one in FeedTest tests the card that was displayed before trying to load a non existing card is not displayed anymore and is replaced by a text (the value of the text is not tested here). And the same way, when we load an existing card, the texts disappears and the real card is shown
* the test in Translation chacks the value of the string in the message

I think these are 2 very different things, this is why I did separate tests in separate files

Reply:
ok